src/js/profile/text.js

```diff
@@ -0,0 +1,20 @@
+import {Component} from "../utils";
+
+export class Text extends Component {
+    constructor(
+        parent,
+        data,
+    ) {
+        super(parent);
+        this.data = data;
+        
+        this.addRawText(data);
+    }
+
+    addRawText = (data) => {
+        $(".bar-chart", this.container).remove();
```

Comment:
where does `this.container` come from?

Reply:
This was used by other code, but not used in this method so I removed `this.container`